test/compile/data/fold.test.ts

```diff
@@ -0,0 +1,64 @@
+/* tslint:disable:quotemark */
+
+import {assert} from 'chai';
+import {FoldTransformNode} from '../../../src/compile/data/fold';
+import {Transform} from '../../../src/transform';
+
+describe('compile/data/fold', () => {
+  describe('Testing FoldTransformNode', () => {
```

Comment:
'Testing FoldTransformNode' => 'FoldTransformNode'

Everything under test is a test. :)